packages/material-ui/test/integration/NestedMenu.test.js

```diff
@@ -1,57 +1,87 @@
 import React from 'react';
-import { assert } from 'chai';
-import { createMount } from '@material-ui/core/test-utils';
-import NestedMenu from './fixtures/menus/NestedMenu';
+import { expect } from 'chai';
+import { cleanup, createClientRender } from 'test/utils/createClientRender';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+function NestedMenu(props) {
```

Comment:
Candidate for improvement. It includes no nested menu as I understand them. There are two sibling menus which is not nesting IMO. Also nitpicking on the fact that the "second-menu" is the first in the tree :smile: 

We probably want to replace this with a test for a cascading menu when we work on https://github.com/mui-org/material-ui/issues/11723